ApplicationLibCode/FileInterface/RifFaultReactivationModelExporter.cpp

```diff
@@ -367,16 +377,20 @@ std::pair<bool, std::string>
 //--------------------------------------------------------------------------------------------------
 std::pair<bool, std::string>
     RifFaultReactivationModelExporter::printSteps( std::ostream&                                                     stream,
-                                                   const std::map<RigFaultReactivationModel::GridPart, std::string>& partNames )
+                                                   const std::map<RigFaultReactivationModel::GridPart, std::string>& partNames,
+                                                   const std::vector<QDateTime>&                                     timeSteps,
+                                                   bool isFirstTimeStepsSelected )
 {
-    int numSteps = 2;
+    // First time step has to be selected in order to export currently
+    if ( !isFirstTimeStepsSelected ) return { false, "Failed to export fault reactivation INP: first time step not selected." };
 
```

Comment:
The timestep selection/validation is being done in the command feature, verifying that the model is valid and all data needed for the modeling is present. This check should be done there, as well, not inside the writer.
The command feature has been updated to do this check (in another branch), so it is not necessary to do this check here.

Reply:
Maybe replace the check with a check that you get at least one timestep?

Reply:
Fixed. Checking if we get at least two time steps.